src/main/java/ch/naviqore/raptor/Raptor.java

```diff
@@ -473,16 +476,25 @@ private static void validateArrivalHandicap(int arrivalHandicap) {
             }
         }
 
-        private int[] validateAndGetStopIdx(Collection<String> stopIds) {
-            return stopIds.stream().mapToInt(this::validateAndGetStopIdx).toArray();
-        }
+        private Map<Integer, Integer> validateStops(Map<String, Integer> stops) {
```

Comment:
Please add a comment above the method which defines a valid stop

Reply:
Addressed in 94f7d96